### PR TITLE
Add initialization scripts and refresh README files

### PR DESCRIPTION
**Pull request by dacspiir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,20 +16,8 @@ You can find Aiia documentation at <https://docs.aiia.eu/>
...
-    ...
-    "ClientId": "your-client-id",
-    "ClientSecret": "your-client-secret",
-    "WebHookSecret": "your-webhook-secret-if-configured"
-  }
-...
-```
-
-Other sections are unnecessary to modify to run the project locally.
+Then run the `initialize.sh` or `initialize.ps1` scripts to setup the project. The script will ask you the `ClientId` and `ClientSecret` you previously configured.
```
Comment (adzspiir) [inline README.md:19]: initialize.sh/ps uses dotnet user-secrets, before we ask to download .NET core. not a big thing, given the audience (probably devs only), but maybe we can shuffle it around a bit?

File: Web/Extensions/WebHostBuilderExtensions.cs [hunk 2]
```diff
@@ -57,15 +57,15 @@ public static IWebHostBuilder UseSerilogElasticSearchIngest(this IWebHostBuilder
             var options = new SiteOptions();
             context.Configuration.Bind(options);
 
-            if (options.Humio.IngestToken.IsSet() && options.Humio.IngestUrl.IsSet())
-                configuration.WriteTo.Elasticsearch(new ElasticsearchSinkOptions(new Uri(options.Humio.IngestUrl))
+            if (options.ElasticSearch.IngestToken.IsSet() && options.ElasticSearch.IngestUrl.IsSet())
+                configuration.WriteTo.Elasticsearch(new ElasticsearchSinkOptions(new Uri(options.ElasticSearch.IngestUrl))
```
Comment (adzspiir) [inline Web/Extensions/WebHostBuilderExtensions.cs:61]: let's add a reminder somewhere to add these to keyvault

File: initialize.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$ROOT/Web"
+
+echo "Envirnoment configuration script."
```
Comment (adzspiir) [inline initialize.sh:5]:
```suggestion
echo "Environment configuration script."
```

File: README.md (hunk 1 above)
Comment (adzspiir) [reply README.md:19]: also - at least in the PS script - we start with `cd Web`, so maybe we can just add a stronger emphasis that the script needs to be run from the root of the project?

File: Web/Extensions/WebHostBuilderExtensions.cs (hunk 2 above)
Comment (dacspiir) [reply Web/Extensions/WebHostBuilderExtensions.cs:61]: I added a comment on the ticket. There will be many other things to fix for sure.

**Review by adzspiir — APPROVED** [review]

**Pull request merged**